public/index.html

```diff
@@ -0,0 +1 @@
+<div id="root"></div>
```

Comment:
Not sure how you are running HW3 without an html.  I wasn't able to get it running in live server.